packages/fxa-auth-server/lib/senders/emails/templates/verifyLogin/index.txt

```diff
@@ -1,10 +1,10 @@
-verifyLogin-title = "New sign-in to <%- clientName %>"
+verifyLogin-title-2 = "Did you sign in to <%- clientName %>?"
 
-verifyLogin-description = "For added security, please confirm this sign-in from the following device:"
+verifyLogin-description-2 = "Help us keep your account safe by confirming you signed in on:"
 
 <%- include('/partials/userInfo/index.txt') %>
 
-verifyLogin-action = "Confirm sign-in:"
+verifyLogin-action = "Confirm sign-in"
 <%- link %>
 
 <%- include('/partials/changePassword/index.txt') %>
```

Comment:
Please replace `changePassword` partial with `automatedEmailNoAction`, as MJML and TXT currently do not match. Thanks!

<img width="899" alt="Screen Shot 2022-08-15 at 11 49 48 AM" src="https://user-images.githubusercontent.com/28129806/184669164-71164cb1-ed18-4307-a0a9-a08977062e76.png">